src/moveFile.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function copyFileContentAndMove(source, destination) {
+  try {
+    const fileContent = fs.readFileSync(source, (err) => {
+      if (err) {
+        return err;
+      }
+    });
+
+    fs.writeFileSync(destination, fileContent, (err) => {
+      if (err) {
+        return err;
+      }
+    });
+
+    fs.unlink(source, (err) => {
+      if (err) {
+        return err;
+      }
+    });
+  } catch (error) {
+    throw error;
+  }
+}
+
+function moveFile(command, source, destination) {
```

Comment:
Consider validating if the source file exists before proceeding with the move operation.

Reply:
Done